homeassistant/components/dlna_dmr/__init__.py

```diff
@@ -1 +1,56 @@
 """The dlna_dmr component."""
+from __future__ import annotations
+
+from homeassistant import config_entries
+from homeassistant.components.media_player.const import DOMAIN as MEDIA_PLAYER_DOMAIN
+from homeassistant.const import CONF_PLATFORM, CONF_URL
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN, LOGGER
+
+PLATFORMS = [MEDIA_PLAYER_DOMAIN]
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up DLNA component."""
+    if MEDIA_PLAYER_DOMAIN not in config:
+        return True
+
+    for entry_config in config[MEDIA_PLAYER_DOMAIN]:
```

Comment:
This way of importing platform config is not robust. It doesn't catch all accepted ways of writing the YAML for platforms. We should import the platform config from `async_setup_platform` in the media_player platform instead.